libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboard.tsx

```diff
@@ -220,37 +221,42 @@ export class VisionExplanationDashboard extends React.Component<
   ): void => {
     this.setState({ searchValue: newValue || "" });
   };
-  private onItemSelect = (item: IVisionListItem): void => {
-    this.setState({ panelOpen: !this.state.panelOpen, selectedItem: item });
-    const index = item.index;
+  private onItemSelect = (item: IVisionListItem, selectedObject = -1): void => {
+    this.setState({ panelOpen: true, selectedItem: item });
     const { computedExplanations, loadingExplanation } = this.state;
-    const computedExplanation = computedExplanations.get(index);
-    if (computedExplanation) {
-      loadingExplanation[index] = false;
-      this.setState({
-        loadingExplanation
-      });
-      return;
+    if (selectedObject !== -1) {
+      if (computedExplanations.get(item.index)?.get(selectedObject)) {
+        loadingExplanation[item.index][selectedObject] = false;
```

Comment:
looks like some code scanning tool is failing here

Reply:
discussed offline, will ignore the warning